content/200-concepts/100-components/02-prisma-client/230-handling-exceptions-and-errors.mdx

```diff
@@ -6,13 +6,6 @@ metaDescription: 'This page covers how to handle exceptions and errors'
 
 <TopBlock>
 
-<Admonition type="warning">
-
-**Error handling incomplete in MongoDB** <br />
-Error handling for the [MongoDB connector](/concepts/database-connectors/mongodb) is incomplete.
-
-</Admonition>
```

Comment:
I remember this being added very early on before it got GA'ed. I think there are still some details that could be polished (eg: some mongo errors may _not_ be returned granularly enough), but this can be removed.

Reply:
Thanks for the clarification. 🙏